azure-pipelines.tools.yml

```diff
@@ -0,0 +1,12 @@
+steps:
```

Comment:
Can you add a comment about what this pipeline does?

Reply:
The pipeline strategy is in the PR description, but I can add something to this file as well.